paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetContract.kt

```diff
@@ -66,6 +66,7 @@ class PaymentSheetContract :
                 },
                 config = config ?: PaymentSheet.Configuration.default(context),
                 statusBarColor = statusBarColor,
+                initializedViaCompose = false,
```

Comment:
This _could_ be used in Compose via `rememberLauncherForActivityResult`.

Reply:
I'm okay with that. But thanks for calling it out!